superset/viz.py

```diff
@@ -1854,6 +1854,8 @@ def query_obj(self) -> QueryObjectDict:
                 _("Pick exactly 2 columns as [Source / Target]")
             )
         qry["metrics"] = [self.form_data["metric"]]
+        if self.form_data.get("sort_by_metric", False):
```

Comment:
you don't need the False here, it'll default to None which is falsey

Reply:
I thought supplying a default helps indicating this parameter is a boolean.

Since CI is green, I think I'll leave it.